assets/externalized/army-garrison-groups.json

```
@@ -0,0 +1,66 @@
+/**
+ * Defining garrison groups (stationary enemy troops) that should present at game start.
```

```suggestion
 * Defining garrison groups (stationary enemy troops) that should be present at game start.
```
Can't believe Canadians make so many typos.